source/dub/generators/build.d

```diff
@@ -82,8 +82,10 @@ class BuildGenerator : ProjectGenerator {
 		enforce(!(settings.rdmd && root_ti.buildSettings.targetType == TargetType.none),
 				"Building package with target type \"none\" with rdmd is not supported yet.");
 
-		logInfo("Performing \"%s\" build using %s for %-(%s, %).",
-			settings.buildType, settings.platform.compilerBinary, settings.platform.architecture);
+		logInfo("Starting", Color.green,
```

Comment:
I think we should use light green for unfinished positive things and only use the darker green for finished things.